druid/src/app.rs

```diff
@@ -187,7 +189,7 @@ impl Default for WindowConfig {
     fn default() -> Self {
         WindowConfig {
             size: None,
-            min_size: None,
+            min_size: Some(WINDOW_MIN_SIZE),
```

Comment:
You also need to change the type definition on line 44 and the function on line 236.